### DISCUSSION TITLE
Can we add to bounties for Issues?

### DISCUSSION THREAD

**Discussion by AelzarBay** [rustdesk/rustdesk#6342] [Q&A] [answered]
I saw the devs put up a $100 bounty for the Android Issue; can users like me add money to the bounty to entice more attention? Is there some way regular users can add to the bounty?

**Comment by rustdesk** [comment, marked as answer]
Yes, you can, https://github.com/rustdesk/rustdesk/issues/3556#issuecomment-1761395027
